yt_dlp/extractor/rokfin.py

```diff
@@ -21,6 +26,9 @@
 
 class RokfinIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?rokfin\.com/(?P<id>(?P<type>post|stream)/\d+)'
+    _NETRC_MACHINE = 'rokfin'
+    _TOKEN_DISTRIBUTION_POINT_URL_STEP_6_7 = 'https://secure.rokfin.com/auth/realms/rokfin-web/protocol/openid-connect/token'
```

Comment:
best to move inside the login function imo

Reply:
I don't know how I could move `_TOKEN_DISTRIBUTION_POINT_URL_STEP_6_7` into `_perform_login` since `_TOKEN_DISTRIBUTION_POINT_URL_STEP_6_7` is being shared by  `_perform_login` and `_download_json_using_access_token`.